zerver/lib/push_notifications.py

```diff
@@ -561,6 +561,7 @@ def send_notifications_to_bouncer(
         # user_uuid is the intended future format, but we also need to send user_id
         # to avoid breaking old mobile registrations, which were made with user_id.
         "user_id": user_profile.id,
+        "realm_uuid": str(user_profile.realm.uuid),
```

Comment:
Just highlighting this important piece - we have to send the `realm_uuid` to the bouncer to find the right `RemoteRealm` to record the stat

Reply:
Yep, that's expected; we'll need to have that data for auth later too.